src/backend/shared/styles/globals.css

```
@@ -27,6 +27,33 @@
+  40% {
+    opacity: 1;
+  }
+}
+
+.ai-chat-typing-dot {
+  display: inline-block;
+  width: 5px;
+  height: 5px;
+  margin: 0 2px;
+  border-radius: 50%;
+  background: currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)